docs/modules/getting_started/pages/index.adoc

```diff
@@ -0,0 +1,18 @@
+= Getting started
+
+This guide will get you started with Apache Hive using the Stackable Operator. It will guide you through the installation of the Operator and its dependencies, setting up your first Hive metastore instance and connecting to it, ingesting example data and querying that data.
```

Comment:
```suggestion
This guide will get you started with Apache Hive using the Stackable Operator. It will guide you through the installation of the Operator and its dependencies and setting up your first Hive metastore instance
```

Reply:
done